honeybee_vtk/hbjson.py

```diff
@@ -0,0 +1,231 @@
+"""Functions to extract geometry and metadata from a valid HBJSON."""
+
+from .helper import get_mesh_points, check_convex
+from .helper import get_point3d, get_end_point
+
+
+def get_data(hbjson_obj, hb_type):
```

Comment:
I added annotations but kept the argument names. I feel it's easy to understand the argument type with these names.

Reply:
Can you point me to where you added these annotations? I can't seem to find them.

Reply:
Isn't HBJSON object from type Dict? Up to you but all this `var_lst` and `var_obj` looks ugly to me. The `obj` one is also confusing because everything in Python is technically an object so you are not really adding useful information here.

Reply:
I revised this. Also, I renamed all the variables with "lst" yesterday.